tests/test_jetpack.py

```diff
@@ -70,7 +70,7 @@ def test_bad_harnessoptions():
 def test_pass_jetpack():
     """Test that a minimalistic Jetpack setup will pass."""
 
-    harnessoptions = {'sdkVersion': '1.8-dev',
+    harnessoptions = {'sdkVersion': '1.17',
```

Comment:
The tests use `tests/resources/bootstrap.js`, which is jetpack 1.17 now (was 1.8-dev before), therefore all those need to be updated, otherwise the tests fail.